sample/benchmark/src/main/java/com/datadog/benchmark/sample/ui/LogPayloadUtils.kt

```diff
@@ -18,6 +25,27 @@ internal fun LogPayloadSize.createLogAttributes(): Map<String, Any?> {
     }
 }
 
+internal val ALL_LOG_LEVELS = listOf(
+    Log.ERROR,
+    Log.VERBOSE,
+    Log.ASSERT,
+    Log.WARN,
+    Log.INFO,
+    Log.DEBUG
+)
+
+internal fun Int.stringRepresentation(): String {
```

Comment:
Let's make function name more specific, given that receiver type is very generic.

```suggestion
internal fun Int.logStringRepresentation(): String {
```

Reply:
fixed